src/sankeyCircular.js

```diff
@@ -1254,11 +1257,12 @@ import {linkHorizontal} from "d3-shape";
 
               var linkY0AtColumn = py_t - (link.width / 2)
               var linkY1AtColumn = py_t + (link.width / 2)
+              var dy
```

Comment:
like before, declare the variable at the next higher scope above the if/else statements